SquareRegisterSDK Test App/SCCTestAppDelegate.m

```diff
@@ -66,21 +68,18 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                                    notes:notes
                                                     supportedTenderTypes:supportedTenderTypes
                                                        clearsDefaultFees:clearsDefaultFees
-                                         returnAutomaticallyAfterPayment:NO
+                                         returnAutomaticallyAfterPayment:YES
                                                                    error:&error];
 
     // Perform the request.
-    BOOL const success = [SCCAPIConnection performRequest:request error:&error];
-    if (!success) {
-        UIAlertView *alertView = [[UIAlertView alloc] initWithTitle:@"Register app installed?" message:@"Make sure the register app is installed in the simulator. Log in as the merchant with token 7074ME2C077ZB." delegate:self cancelButtonTitle:@"Dismiss" otherButtonTitles:nil];
-        [alertView show];
-    }
+    [SCCAPIConnection performRequest:request error:&error];
```

Comment:
Discussed offline; these errors are spuriously generated due to calling `performRequest:error` from `application:didFinishLaunchingWithOptions:`.  Will do a more holistic fix at a later date.